services/web-app/layouts/layout/layout.module.scss

```diff
@@ -14,4 +14,7 @@
   // position for PageBreadcrumb
   position: relative;
   margin: $spacing-09 0 0;
+  :global(.cds--side-nav__overlay.cds--side-nav__overlay-active) {
+    z-index: 10;
```

Comment:
How'd you land on 10? Maybe just a personal preference, but I like to add a comment when I have to use a z-index besides -1, 0, or 1.

Reply:
we're currently using 0,1 & 2 in our custom components in the web-app, I just needed a number that was higher than that, figured I'd add some wiggle room.
Happy to change it to lower if you think necessary!
<img width="412" alt="image" src="https://user-images.githubusercontent.com/40550942/153959934-abaa4955-8d04-4a81-a334-5aff4c332ead.png">

Reply:
@francinelucca I'd either go 3 and add a comment why, or maybe use a Carbon z-index. I don't know how/where these are used. https://github.com/carbon-design-system/carbon/blob/48deedf2fb7a343594ed844d6c852c9108a3eb53/packages/components/src/globals/scss/_layout.scss#L14

Reply:
Oh, that's a private mixin. I don't know if Carbon has z-index's as part of the public API. I'd go 3.

Reply:
Oh again, but the function `z` is public. https://github.com/carbon-design-system/carbon/blob/48deedf2fb7a343594ed844d6c852c9108a3eb53/packages/components/src/globals/scss/_layout.scss#L35